doc/requirements.txt

```diff
@@ -3,9 +3,13 @@ appdirs
 autograd
 numpy
 pygments-github-lexers
+git+git://github.com/XanaduAI/pennylane-sf.git#egg=PennyLane_SF
```

Comment:
how come it points to github version?

Reply:
Because the PyPI version doesn't work with the latest GitHub master of PennyLane 😆 Just requires a new release to fix this.